src/types/onyx/NewGroupChatDraft.ts

```diff
@@ -6,6 +6,7 @@ type SelectedParticipant = {
 type NewGroupChatDraft = {
     participants: SelectedParticipant[];
     reportName: string;
+    avatarUri: string;
```

Comment:
```suggestion
    avatarUrl: string;
```
NAB. For consistency